lib/modules/home/services/searching.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:algolia/algolia.dart';
+
+String _generateFilter(
+  Iterable<String> values, [
+  String filter = 'tags',
+  bool and = false,
+]) {
+  final joint = and ? 'AND' : 'OR';
+  final tags = values.map((v) => '$filter:"$v"');
+  return tags.join(' $joint ');
+}
+
+AlgoliaQuery formQuery(AlgoliaIndexReference index, String query) {
+  final tags = <String>[];
+  final words = <String>[];
+  query.split(' ').forEach((e) {
+    if (e.startsWith('#')) {
+      if (e != '#') {
+        tags.add(e.substring(1));
```

Comment:
When the query string is empty, no tags will be added here, and so line 27 (`.filters`) throws an error because the filter string cannot be empty

Reply:
For some reason, this error isn't showing up in the PR preview, but I was getting it locally